test/test-dot.js

```diff
@@ -55,6 +57,70 @@ test('reply.view with dot engine .dot file', t => {
   })
 })
 
+test('reply.view with dot engine .dot file should create non-existent destination', t => {
+  t.plan(3)
+  const fastify = Fastify()
+  const data = { text: 'text' }
+  const engine = require('dot')
+  engine.log = false
+
+  fastify.register(require('../index'), {
+    engine: {
+      dot: engine
+    },
+    options: {
+      destination: 'non-existent'
+    },
+    root: 'templates'
+  })
+
+  t.tearDown(() => rmdirSync('non-existent', { recursive: true }))
```

Comment:
I think cause of this node 10 tests fail, I can remove it I guess but need locally to delete it manually ?